patroni/ctl.py

```diff
@@ -908,6 +1441,36 @@ def get_cluster_service_info(cluster: Dict[str, Any]) -> List[str]:
 
 def output_members(obj: Dict[str, Any], cluster: Cluster, name: str,
                    extended: bool = False, fmt: str = 'pretty', group: Optional[int] = None) -> None:
+    """Print information about the Patroni cluster and its members.
+
+    Information is printed to console through :func:`print_output`, and contains:
+
+        * ``Cluster``: name of the Patroni cluster, as per ``scope`` configuration;
+        * ``Member``: name of the Patroni node, as per ``name`` configuration;
+        * ``Host``: hostname (or IP) and port, as per ``postgresql.listen`` configuration;
+        * ``Role``: ``Leader``, ``Standby Leader``, ``Sync Standby`` or ``Replica``;
+        * ``State``: ``stopping``, ``stopped``, ``stop failed``, ``crashed``, ``running``, ``starting``,
+          ``start failed``, ``restarting``, ``restart failed``, ``initializing new cluster``, ``initdb failed``,
+          ``running custom bootstrap script``, ``custom bootstrap failed``, or ``creating replica``;
```

Comment:
```suggestion
        * ``State``: ``stopping``, ``stopped``, ``stop failed``, ``crashed``, ``running``, ``starting``,
          ``start failed``, ``restarting``, ``restart failed``, ``initializing new cluster``, ``initdb failed``,
          ``running custom bootstrap script``, ``custom bootstrap failed``, or ``creating replica``, and so on;
```

There could be more :)